--- glxext.h.orig	2009-09-11 17:06:19.000000000 +0530
+++ glxext.h	2009-09-30 18:47:25.000000000 +0530
@@ -426,7 +426,9 @@
 
 /* The next two typedefs are placeholders until NVIDIA clarifies these types */
 #ifndef GLX_NV_video_output
-typedef struct { unsigned int dummy; } *GLXVideoDeviceNV;
+/* GtkGLExt replaced the following typedef: */
+/* typedef struct { unsigned int dummy; } *GLXVideoDeviceNV; */
+typedef unsigned int GLXVideoDeviceNV;
 #endif
 
 #ifndef GLX_NV_video_capture
